python/lsst/ts/scheduler/config_schema.py

```diff
@@ -26,7 +26,7 @@
     """$schema: http://json-schema.org/draft-07/schema#
 $id: https://github.com/lsst-ts/ts_scheduler/blob/master/schema/Scheduler.yaml
 # title must end with one or more spaces followed by the schema version, which must begin with "v"
-title: Scheduler v3
+title: Scheduler v4
```

Comment:
Not applying to this line but to this file in general. Please remove all default values from this file. They are ignored now and only add to confusion.